included_software.md

```diff
@@ -9,19 +9,21 @@ For instructions on how to run this image locally to test your build, please see
 The specific patch versions included will depend on when the image was last built (except Ruby). It is highly suggested you depend only on minor versions, so that we can ensure the language has the latest updates (especially if security related).
 
 * Ruby - `RUBY_VERSION`, `.ruby-version`
-  * 2.6.2 (default)
-  * 2.7.2
+  * 2.7 (default)
+  * 2.6
+  * 3.0
   * Any version that `rvm` can install.
 * Node.js - `NODE_VERSION`, `.nvmrc`, `.node-version`
-  * 10 (default)
+  * 16.4 (default)
   * Any version that `nvm` can install.
 * Python - `PYTHON_VERSION`, `runtime.txt`, `Pipfile`
-  * 2.7 (default)
+  * 3.9 (default)
```

Comment:
You mentioned in the PR description that 3.8 is "the default present in the system before installing packages". Does that mean that 3.8 is available automatically without explicitly installing it? If so, I wonder if it still makes sense to stay with 3.8 for default, since it seems like that could reduce startup time? We still have [3 years](https://endoflife.date/python) before it hits EOL, which is almost as long as [Focal itself](https://endoflife.date/ubuntu).

Reply:
I've dug into this a bit and, while there doesn't appear to be any version of Python that's distributed in Ubuntu Focal by default, 3.8 is being installed already as a dependency of `software-properties-common`, which depends on `python3`, which in turn refers to Python 3.8. So, while I was wrong in thinking that it was present in the system *before* installing packages, it is going to be present in the system after installing that package, which we do at the top of the `Dockerfile` (see https://github.com/netlify/build-image/blob/focal/Dockerfile#L19).